src/types.rs

```diff
@@ -268,18 +257,18 @@ impl Default for GlyphCell {
 pub type GlyphCells = Array2<GlyphCell>;
 
 #[derive(Debug, Clone)]
-pub enum RenderItem {
-    Board(Board),
-    Entity(Entity),
+pub enum RenderItem<'a> {
```

Comment:
maybe you could briefly explain again to me how this is more optimized. Which particular function calls become optimal?